chopper_built_value/pubspec.yaml

```diff
@@ -9,20 +9,20 @@ environment:
   sdk: ">=2.1.0 <3.0.0"
 
 dependencies:
-  built_value: ^7.1.0
-  built_collection: ^4.3.2
-  chopper: ^3.0.3
-  http: ^0.12.2
+  built_value: ^8.0.0
+  built_collection: ^5.0.0
+  chopper: ^4.0.0
+  http: ^0.13.0
 
 dev_dependencies:
-  test: ^1.15.4
-  build_runner: ^1.10.3
+  test: ^1.16.4
```

Comment:
Bump it up to version: 1.16.8 ?

Reply:
Done

Reply:
we can probably drop `test_coverage` package since coverage has great support in dart sdk

And I will take care of it after following PR https://github.com/lejard-h/chopper/pull/216